app/forms/totp_setup_form.rb

```diff
@@ -0,0 +1,27 @@
+class TotpSetupForm
```

Comment:
This newly-introduced class looks pretty straightforward, but can we add a simple spec for it?